### DISCUSSION TITLE
make an option to turn on legacy player field

### DISCUSSION THREAD

**Discussion by IdkeMint** [ppy/osu#25244] [Ideas]
i mean like stable 4:3 sized player field (the only reason why i can't play laser it's just disgusting to play 16:9 or something)

**Comment by peppy** [comment]
Can you explain what you mean? This doesn't make much sense to me unfortunately.

**Reply by IdkeMint** [reply]
i feel very uncomfortable playing laser because i can't feel my area in laser for some reason

**Reply by IdkeMint** [reply]
maybe i am wrong about player field resolutions but something is off i can feel it
